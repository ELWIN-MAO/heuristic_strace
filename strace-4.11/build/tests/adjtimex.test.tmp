
mym_waitpid=101138

mym_waitpid=101138

mym_waitpid=101138

syscall_enter_mym_scno=59  sys_name=execve

mym_waitpid=101138

mym_waitpid=101138

syscall_exit_mym_scno=59  sys_name=execve

mym_waitpid=101138

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=101138

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=101138

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=101138

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=101138

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=101138

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=101138

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=101138

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=101138

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=101138

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=101138

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=101138

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=101138

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=101138

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=101138

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=101138

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=101138

syscall_enter_mym_scno=0  sys_name=read

mym_waitpid=101138

syscall_exit_mym_scno=0  sys_name=read

mym_waitpid=101138

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=101138

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=101138

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=101138

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=101138

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=101138

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=101138

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=101138

syscall_enter_mym_scno=158  sys_name=arch_prctl

mym_waitpid=101138

syscall_exit_mym_scno=158  sys_name=arch_prctl

mym_waitpid=101138

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=101138

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=101138

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=101138

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=101138

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=101138

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=101138

syscall_enter_mym_scno=11  sys_name=munmap

mym_waitpid=101138

syscall_exit_mym_scno=11  sys_name=munmap

mym_waitpid=101138

syscall_enter_mym_scno=159  sys_name=adjtimex
adjtimex(
mym_waitpid=101138

syscall_exit_mym_scno=159  sys_name=adjtimex
{modes=0, offset=2768153, freq=1273841, maxerror=209000, esterror=0, status=STA_PLL|STA_NANO|STA_MODE, constant=7, precision=1, tolerance=32768000, time={1489666174, 36115089}, tick=10000, ppsfreq=0, jitter=0, shift=0, stabil=0, jitcnt=0, calcnt=0, errcnt=0, stbcnt=0, tai=0}
mym_scall_rval=0
) = 0 (TIME_OK)

mym_waitpid=101138

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=101138

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=101138

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=101138

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=101138

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=101138

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=101138

syscall_enter_mym_scno=1  sys_name=write

mym_waitpid=101138

syscall_exit_mym_scno=1  sys_name=write

mym_waitpid=101138

syscall_enter_mym_scno=231  sys_name=exit_group

mym_waitpid=101138
+++ exited with 0 +++
