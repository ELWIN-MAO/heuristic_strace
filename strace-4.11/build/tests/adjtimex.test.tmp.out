adjtimex\(\{modes=0, offset=2768153, freq=1273841, maxerror=209000, esterror=0, status=STA_[A-Z]+(\|STA_[A-Z]+)*, constant=7, precision=1, tolerance=32768000, time={1489666174, 36115089}, tick=10000, ppsfreq=0, jitter=0, shift=0, stabil=0, jitcnt=0, calcnt=0, errcnt=0, stbcnt=0, tai=0\}\) = 0 \(TIME_[A-Z]+\)
lo IP Address 127.0.0.1
ens33 IP Address 192.168.159.137
IP Address 127.0.0.1
IP Address 192.168.159.137
